daemon/src/payout_curve.rs

```diff
@@ -1,13 +1,30 @@
 use crate::model::{Leverage, Usd};
 use anyhow::Result;
 use bdk::bitcoin;
+use cfd_protocol::interval::MAX_PRICE_DEC;
 use cfd_protocol::Payout;
 
 pub fn calculate(
-    _price: Usd,
+    price: Usd,
     _quantity: Usd,
-    _maker_payin: bitcoin::Amount,
-    (_taker_payin, _leverage): (bitcoin::Amount, Leverage),
+    maker_payin: bitcoin::Amount,
+    (taker_payin, _leverage): (bitcoin::Amount, Leverage),
 ) -> Result<Vec<Payout>> {
-    Ok(vec![])
+    let dollars = price.try_into_u64()?;
+    let payouts = vec![
+        Payout::new(
```

Comment:
:mega: This API has changed, but you're probably not getting any compiler errors because this file is not included in any module.